test/cloudformation/timestream_cloudformation.yaml

```diff
@@ -0,0 +1,58 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Resources used for monitoring the scale testing runs using Timestream
+Parameters:
+  DatabaseName:
+    Type: String
+    Description: "Timestream database to forward test metrics to"
+  TableName:
+    Type: String
+    Description: "Timestream table to forward test metrics to"
+  BackupDisasterRecoveryRegion:
+    Type: String
+    Default: "us-east-1"
+    Description: "Disaster recovery region for the timestream backup"
+Resources:
+  TimestreamDatabase:
+    Type: AWS::Timestream::Database
+    Properties:
+      DatabaseName: !Sub "${DatabaseName}"
+  TimestreamTable:
+    Type: "AWS::Timestream::Table"
+    Properties:
+      DatabaseName: !Ref "TimestreamDatabase"
+      TableName: !Sub "${TableName}"
+      MagneticStoreWriteProperties:
+        EnableMagneticStoreWrites: true
+      RetentionProperties:
+        MemoryStoreRetentionPeriodInHours: "731" # A full month of memory store retention
```

Comment:
We can enable it for up to a year. I think the only tradeoffs here would be cost of storing large amounts of data in a fast storage solution for longer periods of time. Since we aren't firing a ton of metrics right now, it's probably fine, but I didn't see the utility of us going for looking back, generally, for longer than say 3 months. How does 3 months sound for now?